build/_build/Build.Steps.cs

```diff
@@ -882,7 +882,7 @@ projectPath switch
                     {
                         "Samples.AspNetCoreMvc21" => Framework == TargetFramework.NETCOREAPP2_1,
                         "Samples.AspNetCoreMvc30" => Framework == TargetFramework.NETCOREAPP3_0,
-                        "Samples.AspNetCoreMvc31" => Framework == TargetFramework.NETCOREAPP3_1,
+                        "Samples.AspNetCoreMvc31" => Framework == TargetFramework.NETCOREAPP3_1 || Framework == TargetFramework.NET5_0,
```

Comment:
No `Samples.AspNetCoreMvc50`? 😝

Reply:
It's a reasonable question, tbh, and given we have separate projects for 3.0 and 3.1, maybe we should?🤔

Reply:
Not in this PR of course. To the backlog!